### PR TITLE
added provider Latch

### PR DESCRIPTION
**Pull request by ffr4nz**
Added Latch service: https://latch.elevenpaths.com

ruby verify.rb 
No errors. You're good to go!

### CODE REVIEW

**Comment by Carlgo11** [comment]
No documentation?

**Comment by jdavis** [comment]
I restarted the build for you.

File: _data/security.yml [hunk 1]
```diff
@@ -72,3 +72,8 @@ websites:
       tfa: Yes
       software: Yes
       doc: http://support.tinfoilsecurity.com/customer/portal/articles/1559364-do-you-support-two-factor-authentication-how-do-i-set-it-up-
+
+    - name: Latch
+      url: https://latch.elevenpaths.com
+      img: latch.png
+      software: Yes
```
Comment (Carlgo11) [inline _data/security.yml]: If the site supports tfa you need to both add the way it uses tfa (such as software, sms, email etc) and set `tfa: Yes`

**Comment by ffr4nz** [comment]
Thanks guys! I've added tfa and doc fields!

**Comment by Carlgo11** [comment]
@ffr4nz That doc link doesn't show any documentation about how to enable TFA.

**Comment by ffr4nz** [comment]
@Carlgo11 my fault, wrong place. Added Latch to providers

**Comment by mxxcon** [comment]
@ffr4nz Last I checked `L` was not after `Y` :wink:

File: _data/providers.yml [hunk 2]
```diff
@@ -210,3 +210,8 @@ providers:
     img: yubico.png
     hardware: Yes
     software: Yes
+
+  - name: Latch
+    url: https://latch.elevenpaths.com
+    img: latch.png
+    software: Yes
```
Comment (jdavis) [inline _data/providers.yml]: He means you should alphabetize the list within the Yaml =]

Right now you have Latch after Yubico.

Comment (ffr4nz) [inline _data/providers.yml]: Thanks! Done!

File: _data/providers.yml [hunk 3]
```diff
@@ -61,6 +61,11 @@ providers:
     img: google.png
     software: Yes
 
+  - name: Latch
+    url: https://latch.elevenpaths.com
+    img: latch.png
+    software: Yes
+
```
Comment (Walkman100) [inline _data/providers.yml:68]: You don't have the `doc:` and `tfa:` field.

Comment (ffr4nz) [inline _data/providers.yml:68]: As I can see providers doesn't have this fieds. Commit 714b585 was a mistake, pull request just include changes in providers.yaml.

Comment (Walkman100) [inline _data/providers.yml:68]: facepalm
[New Providers](https://github.com/jdavis/twofactorauth/blob/master/CONTRIBUTING.md#new-providers)
Sorry. My fault, didn't check that.

Comment (ffr4nz) [inline _data/providers.yml:68]: Don't worry. Pull request is being a mess

**Comment by jamcat22** [comment]
:+1:

**Pull request merged**